packages/wonder-blocks-modal/components/one-column-modal.js

```diff
@@ -0,0 +1,119 @@
+// @flow
+import * as React from "react";
+import {StyleSheet, css} from "aphrodite";
+
+import Color from "wonder-blocks-color";
+import {View} from "wonder-blocks-core";
+import {Grid, Row, Cell, FlexCell, GRID_MODAL_SPEC} from "wonder-blocks-grid";
+
+import ModalCloseButton from "./modal-close-button.js";
+import ModalFooter from "./modal-footer.js";
+
+/**
+ * A one-column modal layout.
+ */
+export default class OneColumnModal extends React.Component<{
+    /**
+     * Called when the close button is clicked.
+     *
+     * If you're using `ModalLauncher`, you probably shouldn't use this prop!
+     * Instead, to listen for when the modal closes, add an `onClose` handler
+     * to the `ModalLauncher`.
+     *
+     * This defaults to a no-op via `defaultProps`. (When used in a
+     * `ModalLauncher`, we'll automatically add an extra listener here via
+     * `cloneElement`, so that the `ModalLauncher` can listen for close button
+     * clicks too.)
+     */
+    onClickCloseButton: () => void,
+
+    /** The footer to show under the contents. */
+    footer?: React.Node,
+
+    /** The contents of the modal. */
+    content: React.Node,
+}> {
+    static defaultProps = {
+        onClickCloseButton: () => {},
+    };
+
+    render() {
+        const {footer, content} = this.props;
+
+        return (
+            <View style={styles.container}>
+                <Grid spec={GRID_MODAL_SPEC}>
+                    <Row flush>
```

Comment:
Nice to see this in the examples.

Reply:
I was playing around with this property and deselecting it didn't do what I expected it to do.  With `flush` I assumed that it would have no gutter, but instead it has no gutter with extra padding.

Reply:
Heya! I'm curious if this matches what I have explaining the `flush` prop in the docs? "Make the cell contents flush with the row. Removes the margins and gutters and adds their spacing as paddings to the cells. Margin spacing gets added as padding to the first and last cell inside the row. Gutter spacing gets added to the right-hand-side of all cells, except for the last one, as padding."